resources/sass/layouts/_breadcrumbs.scss

```diff
@@ -0,0 +1,52 @@
+.ris-breadcrumbs {
+    display: inline-flex;
+    list-style: none;
+    padding: 0;
+
+    @include breakpoint($bp-mobile-max) {
+        margin: $indent-m;
+    }
+
+    @include breakpoint($bp-tablet-min) {
+        margin: $indent-m $indent-ml;
+    }
+
+    &__home-link {
+        line-height: 1;
+
+        &::before {
+            color: $c-secondary;
+            content: 'c';
```

Comment:
better to use separate element for icons, for example <span class="..."></span> and not redefine icon on the whole component with :after or :before